lib/srv/regular/sshserver.go

```diff
@@ -787,6 +790,13 @@ func New(addr utils.NetAddr,
 		SessionRegistry: s.reg,
 	}
 
+	if len(s.awsMatchers) != 0 {
+		s.ec2Watcher, err = server.NewCloudServerWatcher(s.awsMatchers)
```

Comment:
The function call does not match with the signature (missing context and clients)